src/WebToPay/Util.php

```diff
@@ -29,6 +32,34 @@ public function encodeSafeUrlBase64($text) {
         return strtr(base64_encode($text), array('+' => '-', '/' => '_'));
     }
 
+
+    /**
+     * Decrypts string with aes-256-gcm algorithm
+     *
+     * @param $stringToDecrypt string
+     * @param $key string
+     *
+     * @return string|false
```

Comment:
Maybe better return string|null? [We always return value of one type. Optionally, we can return null when using any other return type, too.](https://github.com/paysera/php-style-guide#return-types)